src/protocol-analyzer.cc

```diff
@@ -63,7 +64,9 @@ void ProtocolAnalyzer::Process(bool is_orig, int len, const u_char* data) {
         hilti::rt::context::CookieSetter _(&endp->cookie());
         endp->process(len, reinterpret_cast<const char*>(data));
     } catch ( const spicy::rt::ParseError& e ) {
-        reporter::weird(endp->cookie().analyzer->Conn(), e.what());
+        STATE_DEBUG_MSG(is_orig, hilti::rt::fmt("parse error, triggering analyzer violation: %s", e.what()));
```

Comment:
Let's also remove all the `reporter::weird` functions since they are now unused.